### DISCUSSION TITLE
NeteaseCloudMusicApi停止维护

### DISCUSSION THREAD

**Discussion by L7132** [SPlayer-Dev/SPlayer#151] [讨论 General]
### 是网页端还是客户端

网页端

### 当前系统环境

win10

### 当前 Node.js 及 npm 版本

v16

### 当前版本

v2.0.2

### 具体信息

https://github.com/Binaryify/NeteaseCloudMusicApi
保护版权,此仓库不再维护

**Comment by imsyy** [comment]
哎呀，这下完蛋了 😭

**Comment by youpingfang** [comment]
这个用的真不错，可惜了你的项目。离谱

**Comment by crisqyxw** [comment]
https://github.com/GuihongWang/NeteaseCloudMusicApi
似乎冒出来一个别的Repo，不知道能不能持续下去（

**Comment by imsyy** [comment]
> https://github.com/GuihongWang/NeteaseCloudMusicApi 似乎冒出来一个别的Repo，不知道能不能持续下去（

不好说，后续网易云肯定会换加密方式，看维护情况了

**Comment by L7132** [comment]
> 哎呀，这下完蛋了 😭

https://github.com/nooblong/NeteaseCloudMusicApiBackup

**Comment by kirakirai8023** [comment]
作者gitlab仍在更新
[https://gitlab.com/Binaryify/neteasecloudmusicapi](url)
我今天将现在的gitlab文件上传到了github，欢迎fork
[https://github.com/006lp/NeteaseCloudMusicApi](url)
